src/server/scripts/EasternKingdoms/Gilneas/gilneas_chapter_2.cpp

```diff
@@ -621,6 +622,31 @@ class spell_gilneas_worgen_intro_completion : public SpellScript
     }
 };
 
+//Round Up Horse
+class spell_round_up_horse : public SpellScript
+{
+    bool Validate(SpellInfo const* /*spellInfo*/)
+    {
+        if (!sSpellMgr->GetSpellInfo(68903)) return false;
+        return true;
+    }
+
+    void HandleEffectDummy(SpellEffIndex /*effIndex*/)
+    {
+        if (GetHitUnit()->GetTypeId() != TYPEID_UNIT ||
+            GetCaster()->GetTypeId() != TYPEID_PLAYER ||
+            GetCaster()->ToPlayer()->GetQuestStatus(14416) !=
```

Comment:
EN: that conditional does not make sense, with GetHitUnit()->GetTypeId() != TYPEID_UNIT then according to that logic it will cast the spell to any npc, I do it now this way and it would save me many of the conditions that you did in the sql.
ES: ese condicional no tiene sentido, con GetHitUnit()->GetTypeId() != TYPEID_UNIT ahi segun esa logica le casteara el spell a cualquier npc, yo lo ahora de esta forma y me ahorraria muchas de las condiciones que hicistes en el sql.

if (GetHitUnit()->GetEntry() == 36555 && GetCaster()->GetTypeId() == TYPEID_PLAYER &&
    GetCaster()->ToPlayer()->GetQuestStatus(14416) != QUEST_STATUS_INCOMPLETE)
    GetHitUnit()->ToCreature()->DespawnOrUnsummon(1);

Reply:
嗯，这样看起来更好！

Reply:
in English please